docs/development.rst

```diff
@@ -223,6 +223,14 @@ and then run in your build directory ::
 
 before running your tests.
 
+A client side test can specify several options, including which plugins should be loaded at the time of the test, for example ::
+
+    add_web_client_test(some_client_test "someSpec.js" ENABLEDPLUGINS "gravatar" "jobs")
```

Comment:
We should change these docs to mention both `PLUGIN` and `ENABLEDPLUGINS` and the behavior of each.